Licensing_Program/config_handling.py

```diff
@@ -194,3 +203,11 @@ def load_configfile(cwd=".", info_level=""):
         )
 
     return config
+
+
+def genearate_default_configfile(cwd="."):
+    """This function generates a simple configuration file
+    with filler values.
+    """
+    with open(CONFIG_FILENAME, 'w') as new_configfile:
+        json.dump(CONFIG_DEFAULT, new_configfile, indent=4)
```

Comment:
Nitpick: Can you add a newline to the end of this line? I think it's required for [pep8](https://www.python.org/dev/peps/pep-0008/).